qiskit_nature/results/electronic_structure_result.py

```diff
@@ -135,24 +135,17 @@ def reverse_dipole_sign(self, value: bool) -> None:
 
     @property
     def total_dipole_moment(self) -> Optional[List[float]]:
-        """Returns total dipole of moment"""
+        """ Returns total dipole of moment """
         if self.dipole_moment is None:
             return None  # No dipole at all
-        tdm: List[float] = []
-        for dip in self.dipole_moment:
-            if np.any(np.equal(list(dip), None)):
-                tdm.append(None)  # One or more components in the dipole is None
-            else:
-                tdm.append(np.sqrt(np.sum(np.power(list(dip), 2))))
-        return tdm
+        dms = [dip for dip in self.dipole_moment[0] if dip is not None]
```

Comment:
What will happen here if `dip is None`? I think in that case the following line may be a problem

Reply:
Sure, I think it is solved now.